services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluLaskentaServiceImpl.kt

```diff
@@ -17,6 +19,7 @@ open class TormaystarkasteluLaskentaServiceImpl(
     override fun calculateTormaystarkastelu(hankeTunnus: String): Hanke {
         // load data with hankeTunnus
         val hanke = hankeService.loadHanke(hankeTunnus) ?: throw HankeNotFoundException(hankeTunnus)
+        hanke.geometriat = geometriatService.loadGeometriat(hanke)
```

Comment:
Pitäisikö tässäkin lentää poikkeus, jos ei ole geometroita jostain syystä?

Reply:
Tai, ehkä ladata geometria vast seuraavan if:in sisällä (se pitäisi olla true vain jos geometriat löytyy..)